optimade/validator/__init__.py

```diff
@@ -39,24 +39,39 @@ def validate():
         ),
     )
     parser.add_argument(
-        "--verbosity", "-v", type=int, default=0, help="The verbosity of the output"
+        "--verbosity",
+        "-v",
+        action="count",
+        default=0,
+        help="""Increase the verbosity of the output.""",
     )
     parser.add_argument(
-        "--as_type",
+        "--as-type",
         "-a",
```

Comment:
It seems this is the convention here? (Small before big.)
```suggestion
        "-a",
        "--as-type",
```

Reply:
To me `-t` looks more appropriate abbreviation instead of `-a` here. Just saying :)